k8s/helm/templates/ingress.yaml

```diff
@@ -13,7 +13,35 @@ metadata:
       ingress.gcp.kubernetes.io/pre-shared-cert: runner
       kubernetes.io/ingress.allow-http: "false"
 spec:
-  backend:
-    serviceName: {{ .Chart.Name }}
-    servicePort: {{ .Values.service.port }}
+  rules:
+  - http:
+      paths:
+      - path: /session/*
+        backend:
+          serviceName: runner
+          servicePort: 80
```

Comment:
the serviceName and servicePort values should still be taken from the helm chart and helm values respectively